pages/common/ghc.md

```diff
@@ -0,0 +1,23 @@
+# ghc
+
+> The Glassgow Haskell Compiler, compiles and links Haskell source files.
+
+- Find and compile all modules in the current directory:
+
+`ghc Main`
+
+- Compile a single file:
+
+`ghc {{file.hs}}`
+
+- Compile using optimization, for faster code:
```

Comment:
I would say the "for faster code" is unnecessary here, and doesn't match the similar entries for other compilers. If we decide to keep this, then the others should be edited to say the same.